charts/policy-controller/Chart.yaml

```diff
@@ -19,6 +19,6 @@ annotations:
   artifacthub.io/license: Apache-2.0
   artifacthub.io/images: |
     - name: policy-controller
-      image: gcr.io/projectsigstore/policy-controller:v1.9.0@sha256:87dffbad3e3d7a7ce9cc4184607240df8f39943db25772e0e4b2800deb7e5134
+      image: ghcr.io/sigstore/policy-controller/policy-controller:v0.2.0@sha256:63cd54b877bcd8b3d6d6b7110955e3f773bd8ad5b16d1b6133d4009c26c6da18
```

Comment:
i changed the repository. It looked weird the image digest to the same repo but pointing to a lower version.